### DISCUSSION TITLE
IS there a dashboard that is SELF HOSTED and can do parallel cypress testing

### DISCUSSION THREAD

**Discussion by sefi-ka** [cypress-io/cypress#28493] [Questions and Help]
and is not sorry cypress

**Comment by portyu9** [comment]
Hi @sefi-ka 

If you specifically mean an **official Cypress dashboard equivalent that you can self-host**, the answer is currently **no**.

Cypress Cloud is not available as a self-hosted product.

Since you also specifically excluded Sorry Cypress, there is no Cypress-provided self-hosted dashboard that gives you the equivalent Cloud orchestration, recorded runs, artifacts, analytics, and automatic spec load balancing.

There is an important distinction, though:

- You can absolutely run Cypress itself on your own CI infrastructure.
- You can also parallelize/shard specs yourself without Cypress Cloud.
- What you do not get from Cypress itself is the self-hosted **dashboard/orchestration service**.

For self-managed parallel execution, community tools such as `cypress-split` or `cypress-load-balancer` can help distribute specs across CI workers, or your CI provider can shard the spec list directly.

That solves **parallel execution**, but it is not equivalent to a self-hosted Cypress Cloud dashboard.

So if the requirement is specifically **self-hosted dashboard + dynamic orchestration + results/artifacts**, and Sorry Cypress is excluded, you would need another third-party reporting/orchestration product or your own implementation.

Please mark this answer as accepted if it helped, thank you.
